JNA issue in M1 architecture

It seems that due to [this issue](https://github.com/oshi/oshi/issues/1462) M1 architectures are not able to use the plugin, getting the following error

```
Caused by: java.lang.NoClassDefFoundError: Could not initialize class com.github.dockerjava.transport.DomainSocket                                         
        at com.github.dockerjava.okhttp.UnixSocketFactory.createSocket(UnixSocketFactory.java:21)                                                          
        at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:257)                                                               
        at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:183)                                                                     
        at okhttp3.internal.connection.ExchangeFinder.findConnection(ExchangeFinder.java:224)                                                              
        at okhttp3.internal.connection.ExchangeFinder.findHealthyConnection(ExchangeFinder.java:108)                                                       
        at okhttp3.internal.connection.ExchangeFinder.find(ExchangeFinder.java:88)                                                                         
        at okhttp3.internal.connection.Transmitter.newExchange(Transmitter.java:169)                                                                       
        at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:41)                                                            
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:142)                                                               
        at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:117)                                                               
        at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:94)                 
```

Bumping `docker-java-transport-okhttp` fixes the issue

```patch
-    implementation("com.github.docker-java:docker-java-transport-okhttp:3.2.8")
+    implementation("com.github.docker-java:docker-java-transport-okhttp:3.2.14")
```

As a workaround, it's possible to "override" this dependency (as well as commons-lang)

```kotlin
buildscript {
    configurations.classpath {
        resolutionStrategy {
            setForcedModules("org.jooq:jooq-codegen:3.14.3")
        }
    }

    dependencies {
       classpath("com.github.docker-java:docker-java-transport-okhttp:3.2.14")
       classpath("commons-lang:commons-lang:2.6")
    }
}
```

--- a/build.gradle.kts
+++ b/build.gradle.kts
@@ -113,7 +113,7 @@ afterEvaluate {
 dependencies {
     implementation("org.jooq:jooq-codegen:3.14.8")
     implementation("org.glassfish.jaxb:jaxb-runtime:2.3.3")
-    implementation("com.github.docker-java:docker-java-transport-okhttp:3.2.8")
+    implementation("com.github.docker-java:docker-java-transport-okhttp:3.2.14")
     implementation("org.flywaydb:flyway-core:6.4.3")
     implementation("org.zeroturnaround:zt-exec:1.12")
     compileOnly("javax.annotation:javax.annotation-api:1.3.2")

--- a/src/main/java/com/revolut/shaded/org/testcontainers/utility/RegistryAuthLocator.java
+++ b/src/main/java/com/revolut/shaded/org/testcontainers/utility/RegistryAuthLocator.java
@@ -4,7 +4,7 @@ import com.fasterxml.jackson.databind.JsonNode;
 import com.fasterxml.jackson.databind.ObjectMapper;
 import com.github.dockerjava.api.model.AuthConfig;
 import com.google.common.annotations.VisibleForTesting;
-import org.apache.commons.lang.StringUtils;
+import org.apache.commons.lang3.StringUtils;
 import org.slf4j.Logger;
 import org.zeroturnaround.exec.InvalidResultException;
 import org.zeroturnaround.exec.ProcessExecutor;
@@ -18,7 +18,7 @@ import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.TimeUnit;
 import java.util.concurrent.TimeoutException;
 
-import static org.apache.commons.lang.StringUtils.isBlank;
+import static org.apache.commons.lang3.StringUtils.isBlank;
 import static org.slf4j.LoggerFactory.getLogger;
 import static com.revolut.shaded.org.testcontainers.utility.AuthConfigUtil.toSafeString;
 

